report/src/main/webapp/assets/sass/_report.scss

```diff
@@ -138,24 +138,6 @@
         padding-top: 7px;
       }
     }
-
-    i {
```

Comment:
This shouldn't be removed - this is where our classes that determine state of test/url are extended with styles of glyphicon (in our case there should be fontawesome classes instead)